views/elements/current_and_next.tmpl

```diff
@@ -1,17 +1,32 @@
 {{define "current_and_next"}}
-	{{if .Current}}
-		{{template "current_next" .Current}}
-	{{end}}
-	{{if .Next}}
-		{{template "current_next" .Next}}
+	{{with .CurrentAndNext}}
+
+		<div class="row">
+			{{if .Current}}
+				<div class="col-xs-4 col-sm-4 now-next-live" style="background-image:url('/images/index-now-next-live-thumb.jpeg');"></div>
+				<div class="col-xs-8 col-sm-4 now-next-now">
+					<h2>Now</h2>
+					{{template "current_next" .Current}}
+				</div>
+
+			{{end}}
+			{{if .Next}}
+				<div class="col-xs-12 col-sm-4 now-next-next">
+				 	<h2 class="mobile-col-left">Next</h2>
+				 	<div class="mobile-col-right">
+						{{template "current_next" .Next}}
+					</div>
+				</div>
+			{{end}}
+		</div>
+
 	{{end}}
 {{end}}
 
 {{define "current_next"}}
 
-<p>
-	<a href="{{.Url}}">{{.Title}}</a><br>
-	{{.StartTime.Format "15:04"}} - {{.EndTime.Format "15:04"}}
-</p>
+<h3><a href="{{.Url}}">{{.Title}}</a></h3>
```

Comment:
I think since you're linking from the Current and Next boxes, at least the current show should link to the current time slot?

However the time slot I would guess should have enough info about the actual show on it, then link to the full show page for everything else?

Maybe the Next box should link to the normal show? Since it's not going to provide a track list yet?

Having two different links on each (title links to show, time to time slot) may just confuse people? I was thinking of making the whole box for each a link instead?